CnpSdkForNet/CnpSdkForNet/Communications.cs

```diff
@@ -22,6 +22,18 @@ public class Communications
 
         public event EventHandler HttpAction;
 
+        private Dictionary<string, string> _config;
+
+        public Communications(Dictionary<string, string> config)
+        {
+            _config = config;
+        }
+
+        public Communications()
+        {
+            _config = new ConfigManager().getConfig();
+        }
```

Comment:
@vivek42 Is this no-arg constructor okay? Or should it be a blank dictionary?

Reply:
We should leave it there as it is, for backward compatibility.